diff -up credentials-fetcher-1.1.0/api/src/gmsa_service.cpp.boost-fstream credentials-fetcher-1.1.0/api/src/gmsa_service.cpp
--- credentials-fetcher-1.1.0/api/src/gmsa_service.cpp.boost-fstream	2023-02-23 11:12:57.161594199 -0500
+++ credentials-fetcher-1.1.0/api/src/gmsa_service.cpp	2023-02-23 11:13:38.181130607 -0500
@@ -2,6 +2,7 @@
 
 #include <boost/filesystem.hpp>
 #include <credentialsfetcher.grpc.pb.h>
+#include <fstream>
 #include <grpcpp/ext/proto_server_reflection_plugin.h>
 #include <grpcpp/grpcpp.h>
 #include <grpcpp/health_check_service_interface.h>
@@ -180,7 +181,7 @@ class CredentialsFetcherImpl final
 
                         if ( !boost::filesystem::exists( krb_ccname_str ) )
                         {
-                            boost::filesystem::ofstream file( krb_ccname_str );
+                            std::ofstream file( krb_ccname_str );
                             file.close();
 
                             krb_ticket->krb_file_path = krb_ccname_str;
